runtime/lua/vim/inspect.lua

```diff
@@ -288,6 +288,16 @@ function Inspector:putTable(t)
   end
 end
 
+function Inspector:putFunction(f)
+  local info = debug.getinfo(f)
+  self:puts(
+    '<',
+    'function ', self:getId(f), ': ',
+    info.source, ':', info.linedefined,
```

Comment:
maybe we should omit `':', info.linedefined` when it is -1 ?

Reply:
Sure, will do.